### PR TITLE
feat: bring Sanitization directly into header component

### PR DESCRIPTION
**Pull request by Csmith246**
I think we should probably bring something like this directly into the component. Trusting the library user seems like we're leaving it open to be abused if that library user is neglectful

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -66,6 +67,9 @@
         logo-link="https://www.esri.com/en-us/arcgis/products/arcgis-instant-apps/overview"
         info-button="true"
         info-is-open="true"
+
+        custom-header-html="<div>My custom Header test <img onload=alert('hacked'); src='https://picsum.photos/50'/> </div>"
```
Comment (rslibed) [inline src/index.html:71]: Would it be possible to remove the markup from the sample app? Don't think it's necessary to include this in the sample for the component library

File: src/index.html [hunk 2]
```diff
@@ -185,5 +189,12 @@
         }
       });
     </script>
+    <script>
+      setTimeout(()=>{
```
Comment (rslibed) [inline src/index.html:193]: Would it be possible to remove the logic from the sample app? Don't think it's necessary to include this in the sample for the component library

**Review by rslibed — APPROVED** [review]
I agree, I think it's a good idea to run the sanitization within the compnent just to be safe. Good plan :)

**Comment by Csmith246** [comment]
Removed the XSS tests. It would be nice to keep them in some capacity, as like unit tests or something. But let's pursue that later when we're more familiar with testing in Stencil

**Pull request merged**